src/main/java/srijana/workshop_1/Task_1.java

```diff
@@ -1,4 +1,6 @@
 
```

Comment:
Change already exists in PR https://github.com/tpolishchuk/java-core-trainings/pull/14/files, needs to be removed